src/Microsoft.AspNet.Mvc.Razor/RazorPage.cs

```diff
@@ -61,6 +61,12 @@ public HttpContext Context
         /// <inheritdoc />
         public bool IsPartial { get; set; }
 
+        /// <summary>
+        /// Gets the <see cref="IHtmlEncoder"/> to be used for encoding HTML.
+        /// </summary>
+        [Activate]
+        public IHtmlEncoder HtmlEncoder { get; set; }
```

Comment:
Why do you need this? Why not just use the auto injected bits like the HtmlHelper is?